Fixed pom plugin configuration

mdumrauf opened the pull request:
Pom's plugin configuration was broken

Commit:
Fixed pom plugin configuration

Prototik commented:
Mmm... than you need to enable jarsigner plugin for library, demo, builder and translator.

mdumrauf commented:
Ok. I will include it in their poms.

mdumrauf committed:
Enabled maven-jarsigner-plugin in submodules

maven-jarsigner-plugin was added to builder, demo, library and translator projects
